socorro/external/es/crashstorage.py

```diff
@@ -115,29 +116,24 @@ def is_valid_key(key):
     return bool(VALID_KEY.match(key))
 
 
-def remove_bad_keys(data):
```

Comment:
We don't need `remove_bad_keys` anymore because we're only indexing a document with keys that are explicitly specified in super_search_fields.